app/pb_kits/playbook/pb_background/_background.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= content_tag(object.tag,
+    aria: object.aria,
+    data: object.data,
+    id: object.id) do %>
+    <% if object.image_url.present? %>
+      <div class="<%= object.classname %>" style="background-image: url(<%= object.image_url %>)">
```

Comment:
It's important we have a loader. If we can't reuse the image kit we need to add the loader from the image kit to this kit to load background images.

Example to loader with background images:
![](https://miro.medium.com/max/2000/1*QZ5wJuMhLN2iOVnmxu0ZXw.png)

Another option is to use an image but make it position absolute in a container. Pro's you could use the image kit. Downsides could be some z-indexing issues.
Example:
```scss
.img-back {
  width: 100%;
  height: 100%;
  position: absolute;
  z-index: 0; // Optional?
  object-fit: cover;
}
```

Reply:
so we read that if we see "chunk" then lazy load is working, heres what our network lokos like (for both rails and react) is that right? 
<img width="897" alt="Screen Shot 2020-09-29 at 11 23 20 AM" src="https://user-images.githubusercontent.com/64422843/94578955-4b58a900-0246-11eb-8b7a-fb3a62d03d45.png">
we just used the scss from the image kit and added the corresponding class names

Reply:
We added the style `background-size: cover` to the images.